### PR TITLE
Add depth parameter to DOM.getDocument

### PR DESCRIPTION
**Pull request by sarvaje**
Fix ACL issue in windows 10

### CODE REVIEW

File: src/chromeProtocol/dom.ts [hunk 1]
```diff
@@ -880,8 +880,9 @@ module EdgeDiagnosticsAdapter {
             return {}; // actual response to setChildNodes is empty.
         }
 
-        private getDocument(): IWebKitResult {
+        private getDocument(request?: IWebKitRequest): IWebKitResult {
             this.styleSheetAdded(browser.document);
+            var depth = request ? request.params.depth : -1;
```
Comment (dsrodenas) [inline src/chromeProtocol/dom.ts]: This change throws null exception when GetDocument request.params is undefined. Request never must/will be null, it contains id and method at least. What can be null is params when GetDocument does not contains parameters, so please change the line to:
`var depth = request.params ? request.params.depth : -1;`

Comment (sarvaje) [inline src/chromeProtocol/dom.ts]: this method is reused in `line 77` and it is called without parameters, that why i'm checking if `request` exists or not,
anyway I'm going to add also the condition for the params property

**Comment by sarvaje** [comment]
@dsrodenas done!

File: src/chromeProtocol/dom.ts (hunk 1 above)
Comment (dsrodenas) [inline src/chromeProtocol/dom.ts]: Yes you are right, I did not see the reuse in line 77. But the params check is necesary as well.  I'll merge your changes. Thx!!

**Pull request merged**